Core/Core/Serialisation/BaseObjectSerializerV2.cs

```diff
@@ -103,6 +104,7 @@ public string Serialize(Base baseObj)
       }
       return ret;
     }
+    //TODO: handle IReadonlyDictionary
```

Comment:
Personally, I don't mind TODO: comments that aren't tracked.
This is more a comment that the code doesn't handle this case. 
Not that it's a problem, or that it even should. Simply, it currently doesn't.

But up to you, I'd be happy to track this, it will be come important if we ever implement readonly base objects